more explanations on settings

GitMensch opened the pull request:
and back-reference for "show copybooks on hover"

this originates on me wondering why it did not work "as suggested by the README".

Question: Wouldn't it be reasonable for the LSP to have a list of the file names and if there's only a single one to match the copybook for hover/show definition`?
This would lead to most environments working "out of the box".

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/OCamlPro/superbol-studio-oss?pullRequest=367) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/OCamlPro/superbol-studio-oss?pullRequest=367) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/OCamlPro/superbol-studio-oss?pullRequest=367) it.</sub>

GitMensch commented:
requesting review from @nberth

nberth commented on README.md:
```diff
@@ -79,6 +79,13 @@ From here, you can notably configure:
+  a list, where each entry describes a file extension if the copybook name cannot
+  be found as-is (for example `COPY "mycpy.lib"`).
+  In SuperBOL, the `default` option does not correspond to GnuCOBOL's default, but to "cpy" and
+  "cbx" only.
```
Is GnuCOBOL's default list actually `cpy cbl cob` as hinted [here](https://github.com/OCamlPro/gnucobol/blob/fe28973b030f13b34c52d36df6b928069de21f98/cobc/cobc.c#L9077)?

GitMensch replied:
yes, first in upper, then in lower-case

GitMensch commented:
@nberth: any updates on this? I'm not sure if  #368 is likely to be finished soon and have a "frontend" in the vscode extension (it definitely sounds interesting), but with all the other important changes since the last release I'd like to see 0.1.4 going out "rather soon" - and thise one seems like a low hanging fruit before...

nberth commented on README.md:
```diff
@@ -79,6 +79,13 @@ From here, you can notably configure:
+  To configure this setting, you will need to select `Edit in settings.json`. This is
+  a list, where each entry describes a file extension if the copybook name cannot
+  be found as-is (for example `COPY "mycpy.lib"`).
+  In SuperBOL, the `default` option does not correspond to GnuCOBOL's default, but to "cpy" and
+  "cbx" only.
```
```suggestion
  In SuperBOL, the `default` option corresponds to GnuCOBOL's default, which
  is `"cpy"`, `"cbl"`, and `"cob"` (searched in upper-case first, and then in
  lower-case).
```
